sunpy/coordinates/ephemeris.py

```diff
@@ -1,8 +1,11 @@
 """
 Ephemeris calculations using SunPy coordinate frames
 """
+import re
+import tempfile
+
 import numpy as np
-from packaging import version
+import requests
```

Comment:
It seems we don't have an explicit dependancy on requests at the moment? (I would be shocked if we don't have a transitive one.)

We could either add it or use urllib.

Reply:
It looks like `requests` is implicit dependency through `zeep` (and also `sphinx`).  I'd prefer to add `requests` as an explicit dependency because we're going to have it anyway and doing a POST via `urllib` is bit more cumbersome.